plugin/mdip.vim

```diff
@@ -29,12 +29,14 @@ endfunction
 
 function! s:SaveFileTMPWSL(imgdir, tmpname) abort
     let tmpfile = a:imgdir . '/' . a:tmpname . '.png'
+    let tmpfile = substitute(tmpfile, "\/", "\\\\\\", "g")
+    if tmpfile =~ "mnt"
+      let tmpfile = substitute(tmpfile, "\\\\\\\\mnt\\\\\\\\c", "C:", "g")
+    else
+      let tmpfile = '\\\\wsl\$\\Ubuntu'.tmpfile
+    endif
 
-    let clip_command = "Add-Type -AssemblyName System.Windows.Forms;"
-    let clip_command .= "if ([Windows.Forms.Clipboard]::ContainsImage()) {"
-    let clip_command .= "[Windows.Forms.Clipboard]::GetImage().Save(\\\""
-    let clip_command .= tmpfile ."\\\", [System.Drawing.Imaging.ImageFormat]::Png) }"
-    let clip_command = "powershell.exe -sta \"".clip_command. "\""
+    let clip_command = 'powershell.exe -sta "Add-Type -Assembly PresentationCore;\$img = [Windows.Clipboard]::GetImage();\$file = \"'. tmpfile . '\";\$stream = [IO.File]::Open(\$file, \"OpenOrCreate\");\$encoder = New-Object Windows.Media.Imaging.JpegBitmapEncoder;\$encoder.QualityLevel = 90;\$encoder.Frames.Add([Windows.Media.Imaging.BitmapFrame]::Create(\$img));\$encoder.Save(\$stream);\$stream.Dispose()"'
```

Comment:
can you please make it more readable? One-liner is very hard

Reply:
Done